src/test/java/se/iths/jakartaeelab1/dto/PersonDtoTest.java

```diff
@@ -0,0 +1,35 @@
+package se.iths.jakartaeelab1.dto;
+
+import org.junit.jupiter.api.Test;
+import se.iths.jakartaeelab1.dto.PersonDto;
+import se.iths.jakartaeelab1.entity.Person;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class PersonDtoTest {
+    @Test
+    void map_shouldMapPersonToPersonDto() {
+        Person person = new Person();
+        person.setName("John");
+        person.setAge(25);
+        person.setProfession("Developer");
+
+        PersonDto personDto = PersonDto.map(person);
+
+        assertEquals("John", person.getName());
+        assertEquals(25, person.getAge());
+        assertEquals("Developer", person.getProfession());
```

Comment:
Shouldn't the asserts say personDto; and not person; ? As it is right now the asserts does not really test the map-method. It only makes sure the person-object is the same after you mapped it to a personDto-object, which seems a bit unnecessary.

Reply:
if i change it to personDto it stops working, getAge and if i try age says "Cannot resolve method 'getAge' in 'PersonDto'"

Reply:
That is because personDto does not have a method getName(); since it is a Record. Record-classes have the field names as getter-methods. Se suggestion below, I tested it and the test works when changing the method names.
```suggestion
        assertEquals("John", personDto.name());
        assertEquals(25, personDto.age());
        assertEquals("Developer", personDto.profession());
```